.github/workflows/build.yml

```diff
@@ -204,6 +288,21 @@ jobs:
           fi
           echo "TAG=${CI_TAG}" >> $GITHUB_ENV
           echo "BRANCH=${BRANCH}" >> $GITHUB_ENV
+
+      - name: Build image locally
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./build/upgrade/upgrade.Dockerfile
+          push: false
+          load: true
+          platforms: linux/amd64
+          tags: |
+            ${{ env.IMAGE_ORG }}/upgrade:${{ env.TAG }}
+          build-args: |
+            DBUILD_REPO_URL=https://github.com/openebs/upgrade
+            DBUILD_SITE_URL=https://openebs.io
```

Comment:
Are these fields required for images that are built and used locally?

Reply:
No but good to have them as the dockerfile expects these args